app/javascript/components/Waybill.tsx

```diff
@@ -25,7 +26,35 @@ const Waybill = ({ currentUserRole }) => {
       componentMounted.current = false;
     };
   }, []);
-
+  const update_checkpoint_status = (id) => {
+    httpClient.route.get_routes(id).then((response) => setCheckpoints(response.data));
+    // let  new_checkpoints=checkpoints
```

Comment:
Dont leave comments like thats without any clarification. Use Notes like NOTE: or BUGFIX: and short description why u commented some code, or what should be done in this part of code